doc/contributing.rst

```diff
@@ -134,7 +134,16 @@ of the documentation, run::
 
   xdg-open _build/html/index.html
 
+Building Cython code
+--------------------
+
+Part of the code base is written in `Cython`_. To build this code, run::
+
+    ???
```

Comment:
That's the recommended way.

It could be useful to link to the documentation on [debugging](https://cython.readthedocs.io/en/stable/src/userguide/debugging.html) and [profiling](https://cython.readthedocs.io/en/stable/src/tutorial/profiling_tutorial.html) Cython code.

Reply:
Thanks for confirming the recommended way, and the sign-posts to the debugging and profiling documentation!

On the whole Cython seems quite daunting, at least for me, and so maybe here just getting folks started with the basic building of the code would be sufficient I think. And then depending on the scenario people hopefully would find further information and documentation on their own, as and when it's needed, once the code builds.

```suggestion
    pip install -e .[dev]
```